tests/python/gaia-ui-tests/gaiatest/apps/system/regions/search_panel.py

```diff
@@ -59,11 +59,17 @@ def go_to_url(self, url):
 
         Wait(self.marionette).until(lambda m: self.keyboard.is_keyboard_displayed)
         self.keyboard.tap_enter()
-        if 'http' in urllib.quote(url, safe=':/?=&~').lower():
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name in urllib.quote(url, safe=':/?=&~'))
```

Comment:
This never worked in the case of a redirect. When going to an people.mozilla.com, you get redirected to people.mozilla.org and then we would time out on this wait.